templates/cloud/managed-cloud.html

```diff
@@ -34,7 +34,7 @@
 
 <section class="p-strip--light is-deep is-bordered">
   <div class="row">
-    <div class="col-12">
+    <div class="col-12 u-vertically-spaced--large">
```

Comment:
Vertically-spaced shouldn't be used here. It adds spacing top and bottom when you only need bottom.

Try some of the .u-padding-bottom, maybe .u-padding-bottom--x-large